src/App.js

```diff
@@ -1,15 +1,53 @@
-import React, { useState } from 'react';
+import React, {useState} from 'react';
+import {
+    BrowserRouter as Router,
+    Switch,
+    Route, useHistory
+} from 'react-router-dom';
 import './App.css';
+import TopMenu from './componenten/TopMenu';
+import Blog from './componenten/Blog';
+import Blogpost from './componenten/Blogpost';
+import Login from './componenten/Login';
+
 
 function App() {
-  // We houden in de state bij of iemand is "ingelogd" (simpele versie)
-  const [isAuthenticated, toggleIsAuthenticated ] = useState(false);
-
-  return (
-    <div>
-      Maak hier jouw prachtige blog-applicatie!
-    </div>
-  );
+    // We houden in de state bij of iemand is "ingelogd" (simpele versie)
+    const [isAuthenticated, toggleIsAuthenticated] = useState(true);
+    const history = useHistory();
+
+    function handleClickLogIn() {
+        history.push("/blogposts");
+        toggleIsAuthenticated(true)
+    }
+
+    function handleClickLogOut() {
+        history.push("/");
+        toggleIsAuthenticated(false)
+    }
+
+    return (
+        <>
+            <TopMenu/>
+            <Switch>
+                <Route exact path="/">
+                    <p>Welcome to this Blog</p>
+                </Route>
+                <Route path="/Blog">
```

Comment:
@jamiebloem: voor het path hoort "exact' te staan en blog in "lowercase"
`        <PrivateRoute exact path="/blog">
          <Blog/>
        </PrivateRoute>`